packages/cli/src/util/createServer.ts

```diff
@@ -71,7 +71,8 @@ const createSingleProcessPrism: CreatePrism = options => {
 
 async function createPrismServerWithLogger(options: CreateBaseServerOptions, logInstance: pino.Logger) {
   const operations = await getHttpOperationsFromSpec(options.document);
-  await configureExtensionsFromSpec(options.document);
+  const jsonSchemaFakerCliParams = [{ cliOption: 'fillProperties', cliValue: options.jsonSchemaFakerFillProperties }];
```

Comment:
Why an _array_ of objects (`{ cliOption: string; cliValue: any }[]`) here rather than an object (perhaps `{ [option: string]: any }` or something more specific derived from `CreateBaseServerOptions`)?

Reply:
I had originally done some mapping functionality that was made easier with an array, but I was able to refactor that out and didn't think to change the data type to something else. I have updated it to an object as I do prefer that suggestion.